utils/renameKeys.js

```diff
@@ -0,0 +1,10 @@
+const renameKeys = (keysMap, obj) =>
```

Comment:
Could you add a desc please and/or some basic tests in order to know how to use this

Reply:
This method was actually never used in the end so I have removed it completely